aksharamukha/transliterate.py

```diff
@@ -517,6 +518,43 @@ def convert_default(src, tgt, txt, nativize = True, post_options = [], pre_optio
 
     return convert(src, tgt, txt, nativize, pre_options, post_options)
 
+def convert_docx(src, tgt, file, nativize, pre_options, post_options):
+    new_file = io.BytesIO()      
+
+    with ZipFile(file, 'r') as old_archive:
+        with ZipFile(new_file, 'w') as new_archive:
+            for item in old_archive.filelist:
+                if item.filename.startswith('word/') and item.filename.endswith('.xml'):
```

Comment:
an idea how to break the code into chunks that could be more self-explanatory and reduce the deep nesting inside this method: 
```suggestion
                if should_be_converted(item.filename):
                  convert_xml_file(xml_filename)
                else:
                  copy_xml_file(xml_filename)  
```